raiden_contracts/constants.py

```diff
@@ -29,9 +29,9 @@
 CONTRACT_DEPOSIT = "Deposit"
 
 # Timeouts
-TEST_SETTLE_TIMEOUT = 8
+TEST_SETTLE_TIMEOUT = 8 * 15
```

Comment:
Maybe add some comment that this are seconds

Reply:
✅